website/docs/extensions/starknet/contract-target.md

```diff
@@ -90,6 +90,35 @@ openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", br
 build-external-contracts = ["openzeppelin::account::account::Account"]
 ```
 
+## Wildcard support in build-external-contracts
+
+The `[build-external-contracts]` property explained above allows asking Scarb to build contracts from package dependencies as part of proper package build, for example for use in tests. This release adds a possibility to request building many contracts from a module tree at once. For example, the following snippet:
+
```

Comment:
Emphasize, that:
- Each external contract path can use upmost a single wildcard 
- A wildcard has to be the last character in contract path
- Express how matching is conducted - Scarb will take the contract path before wildcard and look for all contracts which paths start with it. 
Please reword appropriately.

Reply:
Alright. I will work on rewording the paragraph.